### PR TITLE
Hot fix

### PR DESCRIPTION
**Pull request by redarqas**

### CODE REVIEW

File: src/main/scala/Test.scala [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Cat(val colour: String)
 
 object HelloWorld {
   def main(args: Array[String]): Unit = {
-    println("Hello, world!")
+    print("Hello, world!")
```
Comment (CharlesD91) [inline src/main/scala/Test.scala]: print is better

File: src/main/scala/Test.scala [hunk 2]
```diff
@@ -3,9 +3,9 @@ class Cat(val colour: String)
...
 
     val c = new Cat("blue")
-    println(c.colour)
+    print(c.colour)
```
Comment (CharlesD91) [inline src/main/scala/Test.scala]: use print

File: src/main/scala/Test.scala (hunk 1 above)
Comment (redarqas) [reply src/main/scala/Test.scala]: Done

File: src/main/scala/Test.scala (hunk 2 above)
Comment (redarqas) [reply src/main/scala/Test.scala]: Done

File: src/main/scala/Test.scala [hunk 3]
```diff
@@ -7,5 +7,6 @@ object HelloWorld {
 
     val c = new Cat("blue")
     println(c.colour)
+    println(c.colour)
```
Comment (CharlesD91) [inline src/main/scala/Test.scala]: this is good thx2

**Pull request merged**